tests/security/atsec/kvm_check.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Run 'kvm check' test case of ATSec test suite
+# Maintainer: xiaojing.liu <xiaojing.liu@suse.com>
+# Tags: poo#101956
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+
+    select_console 'root-console';
+
+    # Install the required packages for libvirt
+    zypper_call('in libvirt');
+    zypper_call('in wget');
+
+    # Upload the network xml file
+    my $file = 'kvm_check_network.xml';
+    assert_script_run 'wget --quiet ' . data_url("atsec/$file");
+
+    # Checked if there is a clean environment
+    if (script_run('virsh net-list --all | grep default') == 0) {
+        assert_script_run('virsh net-undefine default');
+    }
```

Comment:
@Amrysliu I don't think you need undefine default network configuration here :) it is not start by default.

Reply:
It's used to make sure that. If it is not start, `undefine` will not be run.

Reply:
More comments, it if is there, we need double check it is up/online, we need destroy it before undefine?
I suggest you remove these lines or add the destroy logic :)

Reply:
good point. then remove the check